### ISSUE TITLE
How to train the model?

### ISSUE THREAD

**Issue by sunshiding** [koriavinash1/BraTs2018#3]
There are a lot of files in your folder, how should I use it? Is there an order of use?

**Comment by koriavinash1** [comment]
Hi @sunshiding,
This repo was used for all our initial experiments if you wish to finetune the model or train from scratch, please check https://github.com/koriavinash1/DeepBrainSeg/blob/master/examples/finetune.py

the example provides information about fine-tuning, it can be modified and used to train any network from scratch also

**Comment by sunshiding** [comment]
> Hi @sunshiding,
> This repo was used for all our initial experiments if you wish to finetune the model or train from scratch, please check https://github.com/koriavinash1/DeepBrainSeg/blob/master/examples/finetune.py
> 
> the example provides information about fine-tuning, it can be modified and used to train any network from scratch also.

Hi, @koriavinash1,
Thank you, this project is great. I can start to train the model normally. I am using the Tiramisu_3D project, but I have encountered some problems when predicting. In the Tester.py file:

`mask = np.uint8(nib.load(root_path+'get_2D_tiramisu.nii.gz').get_data())`

I did not find the code to generate this data. Looking forward to your answers, thank you very much.

**Comment by koriavinash1** [comment]
removed that, it was just to limit the iterations for 3D network
